components/users/routes.js

```diff
@@ -112,9 +113,27 @@ function usersApi(app) {
           user,
         });
 
+        const query = Object.keys(user);
+
+        const message = query
+          .filter((criteria) => criteria !== 'updatedAt')
+          .map((criteria) => {
+            if (criteria === 'isActive' && user[criteria] == true) {
+              return `User enabled sucessfull`;
+            } else if (criteria === 'isActive' && user[criteria] == false) {
+              return `User disabled sucessfull`;
+            } else {
+              return `Data updated`;
+            }
+          });
+
+        if (message.length == 0) {
+          throw boom.badRequest('Please verify field you want to update');
+        }
```

Comment:
this validation should be before call `usersService` or in first line of method `updateUser`